packages/ui/src/components/va-button/VaButton.vue

```diff
@@ -112,15 +114,11 @@ export default defineComponent({
     })
 
     const textColorComputed = computed(() => {
```

Comment:
2/2 Can be safety removed. As I can see, transparent background successfully handled in useTextColor hook in Alerts and here too.

```suggestion
```